specification/sections/01-flag-evaluation.md

```diff
@@ -163,80 +178,93 @@ FlagEvaluationDetails<MyStruct> myStructDetails = client.getObjectDetails<MyStru
 
 Configuration code includes code to set the provider, instantiate providers, and configure the global API object.
 
-##### Requirement 1.4.10
+#### Requirement 1.4.10
 
 > In the case of abnormal execution, the client **SHOULD** log an informative error message.
 
 Implementations may define a standard logging interface that can be supplied as an optional argument to the client creation function, which may wrap standard logging functionality of the implementation language.
 
-##### Requirement 1.4.11
+#### Requirement 1.4.11
 
 > The `client` **SHOULD** provide asynchronous or non-blocking mechanisms for flag evaluation.
 
 It's recommended to provide non-blocking mechanisms for flag evaluation, particularly in languages or environments wherein there's a single thread of execution.
 
-##### Requirement 1.4.12
+#### Requirement 1.4.12
 
 > In cases of abnormal execution, the `evaluation details` structure's `error message` field **MAY** contain a string containing additional details about the nature of the error.
 
-##### Requirement 1.4.13
+#### Requirement 1.4.13
 
 > If the `flag metadata` field in the `flag resolution` structure returned by the configured `provider` is set, the `evaluation details` structure's `flag metadata` field **MUST** contain that value. Otherwise, it **MUST** contain an empty record.
 
 This `flag metadata` field is intended as a mechanism for providers to surface additional information about a feature flag (or its evaluation) beyond what is defined within the OpenFeature spec itself. The primary consumer of this information is a provider-specific hook.
 
-##### Condition 1.4.14
+#### Condition 1.4.14
 
 > The implementation language supports a mechanism for marking data as immutable.
 
-###### Conditional Requirement 1.4.14.1
+##### Conditional Requirement 1.4.14.1
 
 > Condition: `Flag metadata` **MUST** be immutable.
 
-#### Evaluation Options
+### Evaluation Options
 
-##### Requirement 1.5.1
+#### Requirement 1.5.1
 
 > The `evaluation options` structure's `hooks` field denotes an ordered collection of hooks that the client **MUST** execute for the respective flag evaluation, in addition to those already configured.
 
 See [hooks](./04-hooks.md) for details.
+
+### 1.6. Shutdown
+
+[![experimental](https://img.shields.io/static/v1?label=Status&message=experimental&color=orange)](https://github.com/open-feature/spec/tree/main/specification#experimental)
+
+#### Requirement 1.6.1
+
+> The API **MUST** define a `shutdown` function, which, when called, must call the respective `shutdown` function on the active provider.
+
+The precise name of this function is not prescribed by this specification.
+Relevant language idioms should be considered when choosing the name for this function, in accordance with the resource-disposal semantics of the language in question.
```

Comment:
From a provider's point of view, it would be easier to have guidance on how to name it because I am afraid that it could be hard to keep track on all the different names in all languages.

Reply:
I think this comment might be more valuable on [this PR](https://github.com/open-feature/spec/pull/179). I've based this on those changes, I think they might need to go together. I could combine them without an issue, I just don't want them to get too big. You can see just the diff [here](https://github.com/open-feature/spec/compare/feat/init-and-shutdown...feat/events)